apis/database/v1alpha1/types.go

```diff
@@ -66,7 +66,7 @@ type PostgreSQLInstanceSpec struct {
 	runtimev1alpha1.ResourceClaimSpec `json:",inline"`
 
 	// EngineVersion specifies the desired PostgreSQL engine version, e.g. 9.6.
```

Comment:
This is the code comment mentioned below from which the docs are generated.

Reply:
@hasheddan I think the best approach is setup crossdocs.
So, I tryed to run crossdocs without success... I folow the readme, but a lot of errors appears... 

I think my mistake with crossdocs was in this step: ```Update Gopkg.toml and doc.sh to ensure you're vendoring and running documentation generation for the versions of the Crossplane projects you care about.```

Reply:
I'll follow-up with you offline to get you set up :+1: